engine/source/output/h3d/h3d_results/genh3d.F

```diff
@@ -1084,10 +1115,15 @@ SUBROUTINE GENH3D(TIMERS,X        ,D        ,V           ,A           ,BUFEL
             ENDDO
           ENDIF
 c
-          IF(NSPMD > 1) THEN
+          IF(NSPMD > 1) THEN 
+            if (ispmd == 0)then
+                IPARTSP_P(1:NUMSPHG)=0
+            endif
             CALL STARTIME(TIMERS,MACRO_TIMER_SPMDH3D)
             CALL SPMD_H3D_GATHER_I(IPARTSP,NUMSPH,IPARTSP_P,NUMSPHG)
             CALL STOPTIME(TIMERS,MACRO_TIMER_SPMDH3D)
+            if (ispmd == 0)then
```

Comment:
@OlivierWienholtz 

Some remaining empty loop for print ( without print :-)

Thanks,
Sébastien

Reply:
Done :)